File: projeto/API/PinguinoApp/PinguinoApp.API/Models/Produto.cs
namespace PinguinoApp.API.Models
{
    public class Produto
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Sku { get; set; }
        public string Codigobarras { get; set; }
        public Fornecedor Fornecedor { get; set; }
        public string Descricao { get; set; }
        public decimal Precovenda { get; set; }
        public bool Ativo { get; set; }


    }
}

File: projeto/UI/pinguino/src/app/models/produto.ts
import { Fornecedor } from './fornecedor';
export class Produto {

    constructor() {
        this.id = 0;
        this.nome = '';
        this.sku = '';
        this.codigobarras = '';
        this.fornecedor = new Fornecedor();
            this.descricao = ''
        this.precovenda = 0.0;
        this.ativo = true;
    }

    id: number;
    nome: string;
    sku: string;
    codigobarras: string;
    fornecedor: Fornecedor;
    descricao: string;
    precovenda: number
    ativo: boolean;
}
